<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>advJulia</class>
 <widget class="QDialog" name="advJulia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="reLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>must be filled</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="minX">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>dX x dY:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-weight:600;&quot;&gt;Please be warned&lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt;, it is advised to maintain costant range for both&lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-style:italic;&quot;&gt; X&lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt; and &lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-style:italic;&quot;&gt;Y&lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt; area. Doing otherwise will create a blurred image.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-weight:600;&quot;&gt;Warning &lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt;changing dagree of function will affect performance&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxYLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Max iterations:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>must be filled</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxXLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="reLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Re(c):</string>
   </property>
  </widget>
  <widget class="QPushButton" name="drawButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxIterationsLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>must be filled</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxX">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="minXLineEdit">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="nFunctionCheckBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>function dagree</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxY">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>N:</string>
   </property>
  </widget>
  <widget class="QLabel" name="imLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Im(c)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxIterationsLineEdit_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>must be filled</string>
   </property>
  </widget>
  <widget class="QLabel" name="escLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Escape range:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>70</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>edit area</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="extendedRenderCheckBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>extended render</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>color pattern</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>239</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Horisontal resolution</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertical resolution</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="horisontalRes">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="verticalRes">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Render</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>advJulia</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
